README.md

```diff
@@ -28,19 +28,19 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 ### Prerequisites
 
-Angular CLI requires Node and NPM.  See [.travis.yml](.travis.yml) for the correct versions of Node, NPM and Angular CLI.
-Then make sure Angular CLI has been properly set up.
+Angular CLI requires Node, NPM, Java 8+, and wget.  See [.travis.yml](.travis.yml) for the correct versions of Node, NPM and Angular CLI.
```

Comment:
The .travis.yml only has Java 8. It doesn't have the Node, NPM, and Angular CLI versions.